flask_dance/contrib/nylas.py

```diff
@@ -18,6 +18,7 @@ def make_nylas_blueprint(
     session_class=None,
     storage=None,
     rule_kwargs=None,
+    base_url="https://api.nylas.com/",
```

Comment:
Rather than specifying a `base_url`, which may not point to Nylas at all, I would prefer if this function accepted a `subdomain` argument. The default value could be `None`, which sets the `base_url` to "https://api.nylas.com/". For this use-case, the developer could pass `subdomain="ireland"`, which would result in setting the `base_url` to "https://ireland.api.nylas.com/".

Reply:
@singingwolfboy Thanks for the feedback, I've made the changes. Could you confirm if that's what you had in mind?

Reply:
No, I intended something more like this:
```python
if subdomain:
    base_url = f"https://{subdomain}.api.nylas.com"
else:
    base_url = "https://api.nylas.com"
```

Also, please be sure that you run the tests locally when making changes to this pull request. This pull request will not be merged as long as there are any test failures.